Index: gedit-3.10.4/configure.ac
===================================================================
--- gedit-3.10.4.orig/configure.ac	2014-02-04 10:26:08.225200531 +0000
+++ gedit-3.10.4/configure.ac	2014-02-04 10:26:08.217196532 +0000
@@ -165,9 +165,6 @@
 
 IT_PROG_INTLTOOL([0.50.1])
 
-AM_GNU_GETTEXT_VERSION([0.18])
-AM_GNU_GETTEXT([external])
-
 AC_SUBST([GETTEXT_PACKAGE], [gedit])
 AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], "$GETTEXT_PACKAGE", [Define to the gettext package name.])
 
