Description: Also load plugins from pre-multiarch directory
Author: Jeremy Bicha <jbicha@ubuntu.com>
Forwarded: not-needed
Index: gedit-3.10.4/gedit/gedit-dirs.c
===================================================================
--- gedit-3.10.4.orig/gedit/gedit-dirs.c	2014-02-04 10:26:15.676924460 +0000
+++ gedit-3.10.4/gedit/gedit-dirs.c	2014-02-04 10:26:15.672922461 +0000
@@ -38,6 +38,7 @@
 static gchar *gedit_locale_dir       = NULL;
 static gchar *gedit_lib_dir          = NULL;
 static gchar *gedit_plugins_dir      = NULL;
+static gchar *gedit_fallback_plugins_dir      = NULL;
 static gchar *gedit_plugins_data_dir = NULL;
 
 void
@@ -113,6 +114,9 @@
 	gedit_plugins_dir = g_build_filename (gedit_lib_dir,
 					      "plugins",
 					      NULL);
+	gedit_fallback_plugins_dir = g_build_filename ("/usr/lib/gedit",
+					      "plugins",
+					      NULL);
 	gedit_plugins_data_dir = g_build_filename (gedit_data_dir,
 						   "plugins",
 						   NULL);
@@ -129,6 +133,7 @@
 	g_free (gedit_locale_dir);
 	g_free (gedit_lib_dir);
 	g_free (gedit_plugins_dir);
+	g_free (gedit_fallback_plugins_dir);
 	g_free (gedit_plugins_data_dir);
 }
 
@@ -181,6 +186,12 @@
 }
 
 const gchar *
+gedit_dirs_get_gedit_fallback_plugins_dir (void)
+{
+	return gedit_fallback_plugins_dir;
+}
+
+const gchar *
 gedit_dirs_get_gedit_plugins_data_dir (void)
 {
 	return gedit_plugins_data_dir;
Index: gedit-3.10.4/gedit/gedit-dirs.h
===================================================================
--- gedit-3.10.4.orig/gedit/gedit-dirs.h	2014-02-04 10:26:15.676924460 +0000
+++ gedit-3.10.4/gedit/gedit-dirs.h	2014-02-04 10:26:15.672922461 +0000
@@ -50,6 +50,8 @@
 
 const gchar	*gedit_dirs_get_gedit_plugins_dir	(void);
 
+const gchar	*gedit_dirs_get_gedit_fallback_plugins_dir	(void);
+
 const gchar	*gedit_dirs_get_gedit_plugins_data_dir	(void);
 
 gchar		*gedit_dirs_get_ui_file			(const gchar *file);
Index: gedit-3.10.4/gedit/gedit-plugins-engine.c
===================================================================
--- gedit-3.10.4.orig/gedit/gedit-plugins-engine.c	2014-02-04 10:26:15.676924460 +0000
+++ gedit-3.10.4/gedit/gedit-plugins-engine.c	2014-02-04 10:26:15.672922461 +0000
@@ -108,6 +108,10 @@
 	                             gedit_dirs_get_gedit_plugins_dir (),
 	                             gedit_dirs_get_gedit_plugins_data_dir ());
 
+	peas_engine_add_search_path (PEAS_ENGINE (engine),
+	                             gedit_dirs_get_gedit_fallback_plugins_dir (),
+	                             gedit_dirs_get_gedit_plugins_data_dir ());
+
 	g_settings_bind (engine->priv->plugin_settings,
 	                 GEDIT_SETTINGS_ACTIVE_PLUGINS,
 	                 engine,
